app/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    package="com.illusionware.npsbrowser">

```

Comment:
Can you explain why you removed the `xmlns:tools` and `package` attributes? I'm not very familiar with android development

Reply:
`xmlns:tools` was deleted by mistake, `package` was changed to `namespace` and moved to `app/build.gradle`